File: Sources/Domain/ReminderOption.swift
import Foundation

public enum ReminderOption: Int, CaseIterable, Codable, Sendable {
    case threeDays = 3
    case oneDay = 1
    case sameDay = 0

    public var label: String {
        switch self {
        case .threeDays:
            return "3일 전"
        case .oneDay:
            return "1일 전"
        case .sameDay:
            return "당일"
        }
    }
}

File: android/core/domain/src/main/java/kr/co/cdd/payboard/core/domain/model/ReminderOption.kt
package kr.co.cdd.payboard.core.domain.model

enum class ReminderOption(val daysBefore: Int, val label: String) {
    ONE_DAY(1, "1 day before"),
    THREE_DAYS(3, "3 days before"),
    SEVEN_DAYS(7, "7 days before"),
}
